### DISCUSSION TITLE
Always redirect to back URL when put Authentication ```'middleware'=>['web:check']``` in config file, what I do wrong ?

### DISCUSSION THREAD

**Discussion by Sariln** [opcodesio/log-viewer#43] [Q&A] [answered]
config file : ```'middleware'=>['web:check']```
and it always redirect back when I try to visit route ```/log-viewer```

**Comment by arukompas** [comment, marked as answer]
@Sariln hey, what does the `web:check` middleware do exactly? Does it check for a logged in user? If so, are you actually logged in?

I can't find any examples on the web using `web:check`. Did you mean to write `'middleware' => ['web', 'auth']` instead?

Also, check out the other [authorization options available](https://github.com/opcodesio/log-viewer#authorization) which will give you more control over what user(s) can access the log viewer.
